package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "webpack-dev-server --inline",
-    "test": "karma start --browsers PhantomJS --single-run"
+    "test": "karma start --browsers PhantomJS --single-run",
+    "test-watch": "karma start"
```

Comment:
@linuxonrails has created new configurations, we will use his approach (I think now is in master)